util/helm/helm_test.go

```diff
@@ -110,6 +109,32 @@ func TestHelmDependencyBuild(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestHelm3Template(t *testing.T) {
```

Comment:
Since we made Helm v3 basically the default (binary is named "helm" and v2 was renamed to "helm2"), wouldn't it make sense to make tests also default to v3 (so name functions `TestHelm...()`) and rather explicitly rename functions for testing v2 to `TestHelm2...()`?

Reply:
Agree. Updated all tests to use helm3 and added one helm2 test.